README.md

```diff
@@ -206,16 +206,25 @@ my_function.has_tag("tag1")  # True
 
 ### Disable parts of the schema
 
-You can provide `GPTEnabled` with a list of parameters to ignore, which will be omitted from the schema.
-For example:
+You can provide `GPTEnabled` with a number of settings to selectively disable parts of the schema.
+For example, to omit certain parameters:
 
 ```python
 @GPTEnabled(ignore_parameters=["b", "c"])  # b and c will not be included in the schema
 def my_function(a: int, b: str, c: float):
     # Function code here...
 ```
 
-It is also possible to specify a value for `ignore_parameters` which will be used for all decorated functions
+The available settings are:
+- `ignore_parameters`: A list of parameter names to exclude from the schema (defaults to `[]`).
+- `ignore_all_parameters`: A boolean value indicating whether to exclude all parameters from the schema
```

Comment:
Might be worth noting that this will ignore both parameters above and below it if it's set to true.